### DISCUSSION TITLE
suggestions

### DISCUSSION THREAD

**Discussion by trebr63** [octoshrimpy/quik#84] [Ideas]
it would be nice to have a sound for sending and receiving messages and to make the bubbles less dark

**Comment by octoshrimpy** [comment]
Bubble colors (and customization in general) are on the roadmap, see #92 and  #129. There are a few things I personally find more important, like the resetting of image view scrolling when phone rotates, updating MMS config, getting blocked numbers working correctly, etc. It is coming, though.

**Comment by PhilippeIIAuguste** [comment]
Great app!!!!!

Would be cool to have configurable quick responses such as "yes," "no," "Atlanta," or whatever the user configures.

My preference would be for them to not be predictive/smart, just a list.
